python_modules/dagster/dagster_tests/definitions_tests/auto_materialize_tests/asset_condition_tests/test_asset_condition.py

```diff
@@ -54,3 +57,21 @@ def test_missing_time_partitioned() -> None:
     # if we evaluate from scratch, they're still False
     _, result = state.without_previous_evaluation_state().evaluate("A")
     assert result.true_subset.size == 4
+
+
+def test_serialize_definitions_with_asset_condition():
+    amp = AutoMaterializePolicy(
```

Comment:
nit: This can be `AutoMaterializePolicy.from_asset_condition(...)`, which will be a bit cleaner